composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "silverstripe/cms": "^4.4@dev",
         "silverstripe/admin": "^1.4@dev",
+        "silverstripe/versioned": "^1.3@dev",
```

Comment:
I decided to add an explicit requirements for Versioned. Versioned is implicitly installed because of versioned-admin, but since we're directly referencing Versioned APIs, I think we should explicitly state that the module requires Versioned.